### ISSUE TITLE
Skein Settings in GCODE as a comment at bottom

### ISSUE THREAD

**Issue by bre** [makerbot/ReplicatorG#114]
I'd love it if at the end of the gcode file, I could look at a section that is commented out but has a datadump of all my gcode settings that were used in the slice. Doing this would make it a lot easier for the community to compare settings.

**Comment by kintel** [comment]
I'm not sure it makes sense to share gcode files in general, so nobody would see these settings. I used to do exactly this in the early days of Skeinforge; I wrote the runskeinforge.sh script to deal with this. However, I never ever looked back at the saved settings, mostly because Skeinforge settings are so complex that I wouldn't know how to compare them in a meaningful and structured way.

I'd say: Extract the machine-specific components of Skeinforge and deal with how to tune your makerbot to find the optimal settings -> save away once and for all.
Then: The relevant tunable settings for individual prints should be manageable and would make sense to share (as a config file published with the part in Thingiverse perhaps).
